Client/.env

```diff
@@ -0,0 +1,4 @@
+REACT_APP_AUTH0_DOMAIN=dev-tumjlmbcbam83fd5.us.auth0.com
```

Comment:
.env file should be added in gitignore and not pushed on github.

Reply:
Noted!